images/bootstrap/Dockerfile

```diff
@@ -101,7 +101,7 @@ RUN curl -fsSL https://download.docker.com/linux/$(. /etc/os-release; echo "$ID"
 # Trying to remount these makes for a very noisy error block in the beginning of
 # the pod logs, so we just comment out the call to it... :shrug:
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends docker-ce=5:20.10.* && \
+    apt-get install -y --no-install-recommends docker-ce docker-buildx-plugin && \
```

Comment:
we already dropped `=5.20.10` in KRTE ages ago, pinning this is not meaningful (especially as all versions have been 20.10.x anyhow)